src/backend/joanie/tests/base.py

```diff
@@ -61,3 +61,83 @@ def generate_token_from_user(user, expires_at=None):
             token, the jwt token generated as it should
         """
         return generate_jwt_token_from_user(user, expires_at)
+
+
+class BaseLogTestCase(BaseAPITestCase):
```

Comment:
Could inherit from `TestCase` then we could use multi inheritance in test case where we need to use those two classes (`BaseLogTestCase`, `BaseAPITestCase`) ?

Reply:
Indeed, this was a mistake.